python-lib/azure_translation_api_client.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+"""Module with utility functions to call the Azure Translator API"""
+
+import json
+import logging
+import re
+from typing import AnyStr
+
+import requests
+
+# ==============================================================================
+# CONSTANT DEFINITION
+# ==============================================================================
+
+
+API_EXCEPTIONS = (requests.HTTPError,)
+
+
+# ==============================================================================
+# CLASS AND FUNCTION DEFINITION
+# ==============================================================================
+
+
+class AzureTranslatorClient:
+    """
+    Translation client interfacing with the Azure Translator.
+
+    Args:
+        api_key: Azure Translator API Key
+        location: Azure location, e.g. francecentral
+
+    """
+
+    def __init__(self, api_key, location) -> None:
+        self.api_key = api_key
+        self.location = location
+
+        self.endpoint = "https://api.cognitive.microsofttranslator.com/translate"
```

Comment:
Does this change depending on the client or is it always the same ?
If it is a constant add it the the section above below API_EXCEPTIONS

Reply:
I assume it is always the same as it is used in their official tutorial -- https://docs.microsoft.com/en-us/azure/cognitive-services/Translator/quickstart-translator?pivots=programming-language-javascript&tabs=python#headers

Added it as a constant